### DISCUSSION TITLE
kernel parameter `quiet`

### DISCUSSION THREAD

**Discussion by jwinarske** [OE4T/tegra-demo-distro#189] [Q&A] [answered]
Is there an easy way to append `quiet` to the kernel parameters?  Will this work?
```
KERNEL_ARGS_append = " quiet"
```

**Comment by kekiefer** [comment, marked as answer]
Basically this should work. Note that more recent yocto branches (honister and master) have a new override syntax, so `KERNEL_ARGS:append = " quiet"` would be appropriate in that case. You can do this in your local.conf or in a custom machine conf.

**Reply by kekiefer** [reply]
Also, here's a trick to make this conditional based on the image debug-tweaks setting, so you can make local debug builds that disable this, and have "production builds" be quiet:

```
KERNEL_ARGS:append = " ${@bb.utils.contains('IMAGE_FEATURES', 'debug-tweaks', '', 'quiet',d)}"
```

**Reply by jwinarske** [reply]
I like this approach.  It might be good to have this as standard.  Thanks!
